src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+ul {
+  display: flex;
+  list-style: none;
+}
+
+.moyo_logo {
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  position: relative;
+  left: 0;
+}
+
+.nav__list {
+  text-align: center;
+  justify-content: center;
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/6c39ee3d-fa2d-4210-b513-f38579387f07)
it works only with `display: flex`, but you actually don't need it